frontend/src/metabase/query_builder/components/view/sidebars/QuestionDetailsSidebarPanel.jsx

```diff
@@ -42,14 +41,22 @@ function QuestionDetailsSidebarPanel({
           onOpenModal={onOpenModal}
         />
         <ClampedDescription
-          className="pl1 pb2"
           visibleLines={8}
           description={description}
           onEdit={onDescriptionEdit}
         />
-        <PLUGIN_MODERATION.QuestionModerationSection question={question} />
+        {hasSections && (
+          <BorderedSectionContainer>
+            {isDataset && canWrite && (
+              <DatasetManagementSection dataset={question} />
+            )}
+            <PLUGIN_MODERATION.QuestionModerationSection question={question} />
+          </BorderedSectionContainer>
+        )}
```

Comment:
Sidebar's sections below the description are separated with a divider:

<img src="https://user-images.githubusercontent.com/17258145/140544595-490356c9-6c6d-4ab7-9d01-11af23448396.png" width="50%" height="auto" />

The divider is only shown only if there is a section. Before this PR, the only section that could be displayed here was the `QuestionModerationSection`, so it rendered the border on its own. But now, there is also a dataset management section that should also be separated by a divider, so I moved the border rendering to the sidebar.

The trick here is that the sidebar should know if the moderation plugin is available now to figure out if it should draw a border. The most simple thing I could think of is to add this `hasPlugin: () => true / false` fn to the moderation plugin

Not sure if it's the best solution, would appreciate your feedback

Reply:
I think you could probably use https://developer.mozilla.org/en-US/docs/Web/CSS/:empty to manage this in CSS, since it seems like `hasSections` is only true when the `BorderedSectionContainer` is empty